buildSrc/src/main/resources/gocd-docker-server/cron/cleanup-old-artifacts.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+
+# 14 day retention
```

Comment:
Could we start with a 30 day retention period so that we can keep logs around longer?